storage/sealer/ffiwrapper/sealer_cgo.go

```diff
@@ -24,12 +23,13 @@ import (
 
 	ffi "github.com/filecoin-project/filecoin-ffi"
 	rlepluslazy "github.com/filecoin-project/go-bitfield/rle"
-	commpffi "github.com/filecoin-project/go-commp-utils/ffiwrapper"
-	"github.com/filecoin-project/go-commp-utils/zerocomm"
+	"github.com/filecoin-project/go-commp-utils/v2"
+	"github.com/filecoin-project/go-commp-utils/v2/zerocomm"
 	commcid "github.com/filecoin-project/go-fil-commcid"
 	"github.com/filecoin-project/go-state-types/abi"
 	"github.com/filecoin-project/go-state-types/proof"
 
+	"github.com/filecoin-project/lotus/chain/proofs"
```

Comment:
slight irony of this change in that I originally copied the functions being now referenced in `proofs` from this file in #11581; the logic of this PR is that we're deprecating their use here and we're not using ffi for them so they don't really belong here now